### DISCUSSION TITLE
Advice please parallel inverters - RS485 wiring suggestions / best practice

### DISCUSSION THREAD

**Discussion by Declan1990** [nathanmarlor/foxess_modbus#621] [Q&A] [answered]
Amateur hour level questions but advice on the following would be much appreciated:

Am soon having 2 x H1 (gen2) installed and would like to integrate to an existing raspberry pi I have running home assistant (but I'll be on the learning curve with that too!). The installers have kindly agreed to connect the required wiring during the initial install if I provide the wires etc (saves having to take it apart soon after). 

My understanding is that to see everything I really need to connect to both (rather than just the master). Was looking at the Waveshare RS485 to USB converter. My gut says the most robust way to purchase 2 and wire one inverter to each? (the pi has sufficient spare USB ports). Or is this overkill? 

If connecting both into 1 converter do I combine at the converter or am I best (for instance) to daisy chain through the master? 

Note it says that ethernet cable can be brittle. Not intending to have a long run. If using other cable (and twisting it myself) is there a base level requirement or spec I should be looking for? 

Many thanks in advance!

**Comment by canton7** [comment, marked as answer]
One waveshare and two inverters should be fine. Just make sure you assign different slave ids using the front panel first, or they'll try and talk over each other.

If you're doing short runs, the electrics don't really matter: it's not running at particularly high speed or anything. I use untwisted speaker wire for mine, with a run of 2m or so, and it's rock solid. Similarly it probably doesn't matter how you join the two runs, so whatever's easiest.

(If you're looking at runs of 10s of metres, or even 100s of metres it's a different story, but it'll be probably easier to relocate the Waveshare in that case.)

**Reply by Declan1990** [reply]
Thanks - 1 waveshare purchased ready to be wired up on Monday hopefully
